docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -648,7 +653,6 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
                     showRtl={showRtl}
```

Comment:
I do understand that this is not this PR's problem, but the following markup 

```jsx
<Flex .. >
  <Flex.Item>   <--- no styles here!!
</Flex>
```

looks strange to me. The intent here is just to align rendered items in column - this could be achieved much simpler, with no <Flex.Item> or React Fragment whatsoever:

```
<Flex column>
  <Segment />
  ...
</Flex>
```